stdlib/task/src/test/java/org/ballerinalang/stdlib/task/service/ListenerServiceValidationTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ *
+ */
+
+package org.ballerinalang.stdlib.task.service;
+
+import org.ballerinalang.launcher.util.BCompileUtil;
+import org.ballerinalang.util.exceptions.BLangRuntimeException;
+import org.testng.annotations.Test;
+
+/*
+ * TODO:
+ * We check resources at runtime for now. Hence need to use compile and setup.
+ * If compiler plugin works, replace BCompileUtil.compileAndSetup() with BCompileUtil.compile().
```

Comment:
Can we replace this todo description with github issue? Check all the new TODOs

Reply:
Done